### DISCUSSION TITLE
laravel 9 Sanctum token

### DISCUSSION THREAD

**Discussion by giancarlozucoloto** [quasarframework/quasar#15353] [CLI - SSR mode]
Hi,
I am using laravel 9 sanctum token
the authentication part is all right.
But when I'm inside and I make a request for a route
Protected and returns error 401 unauthorized because when it went through the boot it didn't take time
the token … but if I load the page again the data appears.
Part of the code below:
boot
`const api = axios.create({ baseURL: "http://localhost:8000/api/" });
api.defaults.headers["Authorization"] = `Bearer ${localStorage.getItem(
  "token"
)}`;`
posts.vue
`setup() {
    const posts = ref([])

    onMounted(() => {
      getPosts()
    }
    )

    const getPosts = async () => {
      try {
        const response = await api.get("/blogs")
        posts.value = response.data.data
      } catch (error) {
        alert(error)
      }

    }`

**Comment by metalsadman** [comment]
Have a function that sets it up in the header of your axios instance, then call it within your login function or wherever you prefer.

**Reply by giancarlozucoloto** [reply]
Hello, thank you very much for the feedback.
as I am studying could you send me an example
